schema/attribute/drivers-license.json

```diff
@@ -4,17 +4,26 @@
 	"identityAttribute": true,
 	"identityAttributeRepository": "http://platform.selfkey.org/repository.json",
 	"title": "Drivers License",
-	"description": "A copy of an individuals drivers license.",
+	"description": "Please provide an image of your driver's license. Front and back cover are required.",
```

Comment:
This description is confusing. It asks for **an** image of the drivers's license (Indicating a single image) and then goes on to say that front and back are required. I would be left scratching my head and looking for image editors.

A better option could be:

"Please provide images of the front and back of your driver's license."